### PR TITLE
Updated Proto files (and generated ones)

### PR DESCRIPTION
**Pull request by Ich5003**
Hi,

this pullrequest updates the following:
- Updated the proto files using the given link.
- Regenerated the proto files using a newer version of protoc and protogen for go
- Updated README

### CODE REVIEW

**Comment by AlekSi** [comment]
Hi @Ich5003. It seems that Travis CI is mostly dead now. Will you be able to send a PR to migrate this repo to GitHub Actions?

**Comment by Ich5003** [comment]
Github Actions should work, see .github/workflows/go.yml
I never used Goveralls, maybe you can fix this before merging.

**Comment by Ich5003** [comment]
@AlekSi Has this PR any chances of getting merged?

**Comment by AlekSi** [comment]
@Ich5003 I merged it into a temporary branch to trigger CI. Will do minor tweaks and create a new PR

**Comment by AlekSi** [comment]
https://github.com/qedus/osmpbf/pull/37

File: OSMPBF/README.md [hunk 1]
```diff
@@ -1,19 +1,48 @@
...
+- **File**: osmformat.proto, fileformat.proto
+- **Reason**: Required for generation
+- **Old Code**:
+```protobuf
+option java_package = "crosby.binary";
+```
+- **New Code**:
+
+```protobuf
+option go_package = "OSMPBF";
```
Comment (AlekSi) [inline OSMPBF/README.md:45]: I added this to `Makefile` to make `.proto` files diff smaller:
```sh
protoc --go_out=. \
	--go_opt=paths=source_relative \
	--go_opt=Mfileformat.proto=github.com/qedus/osmpbf/OSMPBF \
	--go_opt=Mosmformat.proto=github.com/qedus/osmpbf/OSMPBF \
	*.proto
```

File: OSMPBF/README.md [hunk 2]
```diff
@@ -1,19 +1,48 @@
...
 ```
+- **Comptatibility**: This changes is expected to be fully compatible with all PBF files.
+
+### Optimization Goal
+- **File**: osmformat.proto, fileformat.proto
+- **Reason**: Better performance
+- **Added Code**:
+
+```protobuf
+option optimize_for = LITE_RUNTIME;
```
Comment (AlekSi) [inline OSMPBF/README.md:30]: It does not affect Go code generator: https://github.com/golang/protobuf/issues/788

**Pull request merged**